lib/Varien/Data/Form/Element/Abstract.php

```diff
@@ -208,7 +208,7 @@ public function removeClass($class)
      */
     protected function _escape($string)
     {
-        return htmlspecialchars($string, ENT_COMPAT);
+        return is_string($string) ? htmlspecialchars($string, ENT_COMPAT) : $string;
```

Comment:
why not casting to string before passing to htmlspecialchars? This way we're sure the method always returns a string. With your change the function will not always return a string

Reply:
You're right. htmlspecialchars on null returned string before. I#ll change.